pkg/apis/storage/fuzzer/fuzzer.go

```diff
@@ -96,6 +96,10 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 					storage.VolumeLifecyclePersistent,
 				}
 			}
+			if obj.Spec.SELinuxMount == nil {
```

Comment:
I think we missed this during the api review for alpha, but do we want validation on the new field similar to other boolean fields? https://github.com/kubernetes/kubernetes/blob/de9ce03f19e8b1ace1e79fae17119820c4232b67/pkg/apis/storage/validation/validation.go#L433

Reply:
I added some unit tests in a new commit, PTAL.